<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDetalles</class>
 <widget class="QDialog" name="dlgDetalles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btOK">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>450</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCancelar">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>450</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="twDetalles">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>271</width>
     <height>391</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btPersona">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>80</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Persona</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btCasa">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>80</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Casa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btLugar">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Lugar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
